src/dynamically-sized-types.md

```diff
@@ -20,7 +20,7 @@ types">DSTs</abbr>. Such types can only be used in certain cases:
   last field, this makes the struct itself a
   <abbr title="dynamically sized type">DST</abbr>.
 
-Notably: [variables], function parameters, [const] and [static] items must be
+Notably: [variables], function parameters, [const], and [static] items must be
```

Comment:
add "items" after "[const]".

Side note: The "Notably:" part is probably from the original document and should either be changed to the Note format used elsewhere in the book or just remove the word and have this be part of the text.

Reply:
> add "items" after "[const]".
> 

Got it!

> Side note: The "Notably:" part is probably from the original document and should either be changed to the Note format used elsewhere in the book or just remove the word and have this be part of the text.

I agree but I can't discern a consistent format methodology for `Note`s. I see they are sometimes in code blocks, sometimes `***Note***` and sometimes `<strong>`. Just let me know what we prefer and I will make it comply!

Thanks :-)

Reply:
Notes shouldn't be in code blocks. Should be `***`, although `<strong>` is equivalent.